soccer-records-model/src/test/java/cz/muni/fi/pa165/team/TeamRepositoryImplTest.java

```diff
@@ -96,14 +96,14 @@ public void findByName()
     @Test
     public void findById()
     {
-        Assert.assertNotNull(teamDao.findTeamById(t2.getId()));
+        Assert.assertNotNull(teamDao.getTeamById(t2.getId()));
```

Comment:
chtělo by to ještě jeden test na to, že to vyhodi vyjimku, viz předchozí komentáře